src/lib/utils/parse.js

```diff
@@ -92,19 +92,28 @@ export const parseModelData = (modelName, modelJson, sites, configProvidedVarian
   const [dates, updated, nowcastFinalDate, dateSummary] = extractDatesFromModels(modelJson)
   const dateIdx = new Map(dates.map((d, i) => [d, i]));
 
+  // Reorder variants so that the pivot is first for all displays
+  const variants = [...modelJson.metadata.variants];
+  // Use the explicit pivot in the metadata if available, otherwise assume the
+  // pivot is the last variant in the array
+  const pivot = modelJson.metadata.pivot || variants[variants.length - 1];
+  const pivotIndex = variants.indexOf(pivot);
+  if (pivotIndex >= 0) {
+    variants.splice(pivotIndex, 1);
+    variants.unshift(pivot);
+  }
```

Comment:
Sorting the pivot variant to the beginning here. 
Current view of the legends + graphs:

![Screenshot 2025-01-31 at 3 46 12 PM](https://github.com/user-attachments/assets/ab7af443-4a33-43bd-8850-3627f43820db)
![Screenshot 2025-01-31 at 3 46 24 PM](https://github.com/user-attachments/assets/08ecae76-548e-4c1b-9b92-6406da70e873)

I'll let @jameshadfield chime in if this the best place in the app to do this 🙏

Reply:
I'd need to revisit this code to properly answer but I think this looks fine and I think we're good to merge this PR